test_project/test_utils/dbt_test_utils.py

```diff
@@ -383,22 +419,75 @@ def context_table_to_csv(self, table: Table, model_name: str) -> str:
 
         return csv_fqn.stem
 
-    def context_table_to_dict(self, table: Table, orient='index'):
+    def context_table_to_dict(self, table: Table, orient='index', use_nan=True):
         """
         Converts a context table in a feature file into a dictionary
             :param table: The context.table from a scenario
             :param orient: orient for df to_dict
+            :param use_nan: Replace <null> placeholder with NaN
             :return: A dictionary modelled from a context table
         """
 
-        table_df = self.context_table_to_df(table)
+        table_df = self.context_table_to_df(table, use_nan=use_nan)
 
         table_dict = table_df.to_dict(orient=orient)
 
         table_dict = self.parse_lists_in_dicts(table_dict)
 
         return table_dict
 
+    def context_table_to_model(self, seed_config : dict, table: Table, model_name: str, target_model_name: str):
+        """
+        Creates a model from a feature file data table
+            :param seed_config: Configuration dict for seed file
+            :param table: The context.table from a scenario or a programmatically defined table
+            :param model_name: Name of the model to base the feature data table on
+            :param target_model_name: Name of the model to create
+            :return: Seed file name
+        """
+
+        feature_data = self.context_table_to_dict(table=table, orient="index", use_nan=False)
+        column_types = seed_config[model_name]["+column_types"]
+
+        sql_command = ""
+        first_row = True
+        first_column = True
+
+        for row_number in feature_data.keys():
+            if first_row:
+                first_row = False
+            else:
+                sql_command = sql_command + "\nUNION ALL \n"
+            first_column = True
```

Comment:
Defined twice, first on line 454

Reply:
Sorry, this is a hangover from PICK / Revelation R, resolved!